base_layer/core/src/proof_of_work/monero_rx.rs

```diff
@@ -37,12 +37,13 @@ use std::{
     fmt::{Display, Error, Formatter},
     iter,
 };
-use tari_crypto::tari_utilities::hex::Hex;
+use tari_crypto::tari_utilities::hex::{from_hex, Hex, HexError};
 use thiserror::Error;
 
-const MAX_TARGET: U256 = U256::MAX;
 pub const LOG_TARGET: &str = "c::pow::monero_rx";
 
+// const MAX_TARGET: U256 = U256::MAX;
```

Comment:
```suggestion
const MAX_TARGET: U256 = U256::MAX;
```
We should use U256::MAX